packages/editor-ui/src/components/NodeCreator/NodeItem.vue

```diff
@@ -83,6 +95,12 @@ export default Vue.extend({
 				left: `${this.draggablePosition.x}px`,
 			};
 		},
+		isCommunityNode(): boolean {
+			return this.nodeType.communityNode === true;
```

Comment:
can simply check directly here using `isCommunityNode` function and we can delete this flag completely